### PR TITLE
Busstop progress indicator

### PR DESCRIPTION
**Pull request by davidread**
Use loopstats to show a progress indicator. e.g.
```
Count: 371,900 Rate: 164,466,000/hour Time: 0:00:08 taken, 0:00:01 remaining. Progress: 86%
```

### CODE REVIEW

File: import_busstops.py [hunk 1]
```diff
@@ -7,6 +7,7 @@ class BusImportCommand(CSVImportCommand):
 
     def __init__(self, *kargs, **kwargs):
         self.dry_run = kwargs.pop('dry_run')
+        kwargs['num_expected_records'] = 430525
```
Comment (andreagrandi) [inline import_busstops.py:10]: where does this number come from?

Comment (davidread) [reply import_busstops.py:10]: That's the number of records I got from the data when running it before.

Comment (andreagrandi) [reply import_busstops.py:10]: ok, but we should pass it as parameter, we don't know if next time we download the Naptan zip files that number will be still the same. Train stations don't change very often but bus stops do, as far as I know

Comment (davidread) [reply import_busstops.py:10]: As long as it is roughly right, it will give an idea of timings. That's all that this is for.

Comment (andreagrandi) [reply import_busstops.py:10]: All right, no problem. If you can just fix merge conflicts I can review it again and merge, cheers

Comment (davidread) [reply import_busstops.py:10]: ta, conflicts sorted now

**Review by andreagrandi — APPROVED** [review]

**Pull request merged**